config/sets/symfony/symfony7/symfony73.php

```diff
@@ -8,4 +8,5 @@
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->import(__DIR__ . '/symfony73/symfony73-console.php');
+    $rectorConfig->import(__DIR__ . '/symfony73/symfony73-twig-bundle.php');
```

Comment:
this needs to be registered in Symfony7SetProvider as well below this:

https://github.com/rectorphp/rector-symfony/blob/4bc47c03887328f3fc886a681f22f232f8280934/src/Set/SetProvider/Symfony7SetProvider.php#L89-L95

Reply:
- https://github.com/rectorphp/rector-symfony/pull/762